feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailFloatingMenu.kt

```diff
@@ -83,22 +82,24 @@ private fun TimetableItemDetailFloatingActionButtonMenu(
     slideUrl: String?,
     videoUrl: String?,
     onExpandedChange: (Boolean) -> Unit,
-    onBookmarkClick: (isBookmarked: Boolean) -> Unit,
+    onBookmarkChenged: (isBookmarked: Boolean) -> Unit,
     onAddCalendarClick: () -> Unit,
     onShareClick: () -> Unit,
     onViewSlideClick: (url: String) -> Unit,
     onWatchVideoClick: (url: String) -> Unit,
     modifier: Modifier = Modifier,
 ) {
     var height by remember { mutableIntStateOf(0) }
-    var childMenuIsBookmarked by remember { mutableStateOf(isBookmarked) } // local copy to update after transition
-    val latestIsBookmarked by rememberUpdatedState(isBookmarked) // to ensure the latest value is used in recomposition
+    var childMenuIsBookmarked by remember(isBookmarked) { mutableStateOf(isBookmarked) } // local copy to update after transition
+    var pendingBookmarkToggle by remember { mutableStateOf(false) }
 
-    // Recompose child menu items only after the view size has settled.
-    // Recomposing them during the transition may cause a brief flicker that makes the menu look flaky.
+    // Synchronize the local copy when the prop changes while collapsed
     LaunchedEffect(height) {
-        delay(100) // small debounce delay to wait until transition stabilizes
-        childMenuIsBookmarked = latestIsBookmarked
+        delay(100)
+        if (pendingBookmarkToggle) {
+            pendingBookmarkToggle = false
+            onBookmarkChenged(!isBookmarked)
```

Comment:
This PR isn’t directly related, but since the `!isBookmarked` being passed here isn’t actually used, it would be really helpful if you could change it to a pure `onBookmarkToggle: () -> Unit` instead.